pm/sender_test.go

```diff
@@ -297,6 +296,7 @@ func TestCreateTicketBatch_UsesSessionParamsInBatch(t *testing.T) {
 		RecipientRandHash: recipientRandHash,
 		ExpirationBlock:   big.NewInt(1),
 		PricePerPixel:     big.NewRat(1, 1),
+		ExpirationParams:  &TicketExpirationParams{},
```

Comment:
Should we have a test that checks for a nil `ExpirationParams` as well as the new conditions?

Reply:
This should actually never be nil in the real case due to the usage of `pmTicketParams()` but I can add a test (see comment above though about the nil check)

Reply:
fixed in c3bcc39f